backend/src/db/migrations/23-video-block-show-link.sql

```diff
@@ -0,0 +1,10 @@
+-- Adds a single `show_link` field to blocks, used for video blocks to also show
+-- a link to the corresponding video page. We also drop and re-add the appropriate constraint.
+alter table blocks
+    add column show_link boolean default false,
```

Comment:
```suggestion
    add column show_link boolean default true,
```

Let's also change that so that all existing video blocks get a link. I think it's fine to do that!